content/en/news/releases/1.21.x/announcing-1.21/change-notes/index.md

```diff
@@ -0,0 +1,326 @@
+
+- **Removed** legacy `EnvoyFilter` implementation for Telemetry. For the majority of users, this change has no impact, and
+was already enabled in previous releases. However, the following fields are no longer respected: `prometheus.configOverride`,
+`stackdriver.configOverride`, `stackdriver.disableOutbound`, `stackdriver.outboundAccessLogging`.
+
+## Extensibility
+
+- **Added** support for outbound traffic using the PROXY Protocol. By specifying `proxyProtocol` in a `DestinationRule` `trafficPolicy`,
+the sidecar will send PROXY Protocol headers to the upstream service. This feature is not supported for HBONE proxy at the present time.
+
+- **Added** support for matching `ApplicationProtocols` in an `EnvoyFilter`.
+
+- **Removed** support for `policy/v1beta1` API version of `PodDisruptionBudget`.
+
+- **Removed** using `BOOTSTRAP_XDS_AGENT` experimental feature to
+apply `BOOTSTRAP` `EnvoyFilter` patches at startup.
+
+## Installation
+
+- **Improved** aborting graceful termination logic if the Envoy
+process terminates early.
+([Issue #36686](https://github.com/istio/istio/issues/36686))
+
+- **Updated** Kiali addon to version v1.79.0.
+
+- **Added** configurable scaling behavior for Gateway HPA in the Helm chart.
+([usage]( https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#configurable-scaling-behavior))
+
+- **Added** `allocateLoadBalancerNodePorts` config option to the Gateway chart.
+([Issue #48751](https://github.com/istio/istio/issues/48751))
+
+- **Added** a message to indicate the default webhook shifting from a revisioned installation to a default installation.
+([Issue #48643](https://github.com/istio/istio/issues/48643))
+
+- **Added** the `affinity` field to Istiod Deployment. This field is used to control the scheduling of Istiod pods.
+
+- **Added** `tolerations` field to Istiod Deployment. This field is used to control the scheduling of Istiod pods.
+
+- **Added** support for "profiles" to Helm installation. Try it out with `--set profile=demo`!
+  ([Issue #47838](https://github.com/istio/istio/issues/47838))
+
+- **Added** the setting `priorityClassName: system-node-critical` to the ztunnel DaemonSet template to ensure it is running on all nodes.
+([Issue #47867](https://github.com/istio/istio/issues/47867))
+
+- **Fixed** an issue where the webhook generated with `istioctl tag set` is unexpectedly removed by the installer.
+([Issue #47423](https://github.com/istio/istio/issues/47423))
+
+- **Fixed** an issue where uninstalling Istio didn't prune all the resources created by custom files.
+([Issue #47960](https://github.com/istio/istio/issues/47960))
+
+- **Fixed** an issue where an installation might be blocked by a "conflicting webhook" error if a previous installation was interrupted or failed.
+([Issue #48112](https://github.com/istio/istio/issues/48112))
+
+- **Fixed** an issue `Makefile.k8s.mk` `fetch-root-ca` if-condition not work and can not handle `cacerts` case.
```

Comment:
Not very good grammar and I see another comment to remove this. Fixing or removing is fine.

Reply:
removed these.